test-resources/linting_formatters_example.clj

```diff
@@ -0,0 +1,47 @@
+(ns linting-formatters-example
```

Comment:
Appreciate much the effort, at the same time are these the API and usage patterns we want to foster?

As I see it, with https://github.com/nedap/formatting-stack/pull/152 all usages should become far more concise, involving less Clojure coding and more option passing.

Probably we would have sensible defaults as well for linting in CI as it's a frequent use case.

Maybe we can just remove this .clj, else it's more stuff to maintain / keep track of (particularly as APIs will fluctuate)

Reply:
I was thinking #152 could showcase its effectiveness by changing this file to the new format. Having extra documentation on old vs new configuration. When this PR is delivered I think this should come with some advice on usage.

Reply:
> I was thinking #152 could showcase its effectiveness by changing this file to the new format. 

Effectiveness is better proven by exercising an alpha for a while

In practice we'd introduce more doc that can confuse a casual user for a while. It's not good to introduce churn, e.g constant API changes or patterns of usage.

Otherwise we complect internal and external goals

Reply:
I added an explanation in the docstring that this config is subject to change. I want to keep the example in, so we have some guidance until #152 or a similar solution is in.